shared/constants/profile.js

```diff
@@ -20,22 +21,42 @@ export type WaitingRevokeProof = TypedAction<'profile:revoke:waiting', {waiting:
 export const finishRevokeProof = 'profile:revoke:finish'
 export type FinishRevokeProof = TypedAction<'profile:revoke:finish', void, {error: string}>
 
+export const updateProofText = 'profile:updateProofText'
+export type UpdateProofText = TypedAction<'profile:updateProofText', {proof: string}, void>
+
+export const updateError = 'profile:updateError'
+export type UpdateError = TypedAction<'profile:updateError', {error: ?string, errorCode: number}, void>
```

Comment:
This is a little weird, there is an action that's just for an error but the error field (in flux standard action) is set to false/undefined.

Maybe we should make this 

`TypedAction<'profile:updateError', void, {error: ?string, errorCode: number}>`